test/unit_tests/test_system_state.c

```diff
@@ -0,0 +1,45 @@
+/*

```

Comment:
This is what unit test files will look like! We can run them with hardware plugged in to verify HAL code, and other small functions. Will come in handy for logger, housekeeping.